.appveyor.yml

```diff
@@ -10,7 +11,7 @@ environment:
 
 install:
     - SET PATH=C:\\Python%PYTHON%;c:\\Python%PYTHON%\\scripts;%PATH%
-    - python -m pip install -U pip wheel setuptools
+    - pip install -U pip wheel setuptools
```

Comment:
Easy fix: `python -m pip.__main__`. That's what `python -m <pkg>` is shorthand for, as it happens. http://stackoverflow.com/a/6630931/166389